### PR TITLE
Add Organizations to main:

### PR DESCRIPTION
**Pull request by omazapa**
Hello guys,
added
Jozef Stefan Institute (JSI)
University of Nebraska Lincoln (UNL)

Addes Participating Organizations to projects:
CERN, JSI -> DD4HEP
CMS -> HahRd - 3DClustering
UNL -> ROOT - RootPackageManager

minor fixes in some projects.

Cheers
Omar.

### CODE REVIEW

File: _gsocproposals/2018/proposal_HAhRD3D-Clustering.md [hunk 1]
```diff
@@ -3,7 +3,9 @@ title: 3D-Clustering
 layout: gsoc_proposal
 project: HAhRD
 year: 2018
-organization: LLR
+organization:
+  - LLR
+  - CMS
```
Comment (etejedor) [inline _gsocproposals/2018/proposal_HAhRD3D-Clustering.md]: CMS is not an organization but a project. Please move it to the project section.

File: _gsocproposals/2018/proposal_HAhRD3D-Clustering.md [hunk 2]
```diff
@@ -1,9 +1,12 @@
 ---
 title: 3D-Clustering
 layout: gsoc_proposal
-project: HAhRD
+project: 
+  - CMS
+  - HAhRD
```
Comment (hegner) [inline _gsocproposals/2018/proposal_HAhRD3D-Clustering.md]: A proposal can be supported by multiple organizations, but only one project. Can you choose either CMS or HAhRD?

File: _gsocproposals/2018/proposal_DD4heptessellation.md [hunk 3]
```diff
@@ -4,7 +4,9 @@ layout: gsoc_proposal
 project:
   - DD4hep
 year: 2018
-organisation: CERN
+organisation: 
+  - CERN
+  - JSI
```
Comment (petricm) [inline _gsocproposals/2018/proposal_DD4heptessellation.md]: Please remove JSI from this proposal

File: _gsocorgs/2018/jsi.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
```
Comment (petricm) [inline _gsocorgs/2018/jsi.md]: And since JSI is not associated with DD4heo this file can also go

**Comment by oshadura** [comment]
@omazapa any news about fate of this PR? I would like to fix my proposal (I am from University of Nebraska, and not from CERN), but I  found that you already added UNL here

**Comment by omazapa** [comment]
Hi @oshadura 

code updated, with the changes.

Cheers,
Omar.

File: _gsocproposals/2018/proposal_DD4heptessellation.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ layout: gsoc_proposal
 project:
   - DD4hep
 year: 2018
-organization:
```
Comment (petricm) [inline _gsocproposals/2018/proposal_DD4heptessellation.md]: This is wrong, and it has to be with a "z"

File: _gsocorgs/2018/jsi.md (hunk 4 above)
Comment (petricm) [reply _gsocorgs/2018/jsi.md]: This file is superfluous since no project is associated with JSI

**Comment by etejedor** [comment]
@omazapa can you check @petricm comments? Thanks!

**Comment by omazapa** [comment]
Hi @etejedor 
done!
Cheers.

**Comment by etejedor** [comment]
Thanks @omazapa for the changes.
@hegner are we happy with this one? There seems to be a failing check but it is not clear to me what the problem is.

**Pull request merged**